index.html

```diff
@@ -10,16 +10,170 @@
+                            Reward
+                        </div>
+                        <div class="content">Defeat the General. Claim glory and grab the great rune. The reward is THE
+                            STARSCOURAGE GREATSWORD!</div>
+                    </div>
+                </div>
+            </div>
+            <div class="underline">SEE THE WHOLE PROGRAM</div>
+        </div>
+    </section>
+    <section id="teachers" class="flex justify-center teachers">
+        <div class="bg">
+            <header class="flex justify-center">
+                <div>
+                    <div>
+                        Featured Teachers
+                    </div>
+                    <div class="flex justify-center">
+                        <div class="divider"></div>
+                    </div>
+                </div>
+            </header>
+            <div id="speakers-container" class="flex justify-center wrap">
+                <!-- js objects go here -->
+            </div>
+        </div>
+    </section>
+    <section id="partners" class="flex justify-center partners">
+        <div>
+            <div class="text-center" style="color: #999999; font-size: 30px; margin-top: 30px;">
+                Partners
+            </div>
+            <div class="flex justify-center">
+                <div class="divider"></div>
+            </div>
+            <div class="flex justify-center wrap">
+                <img src="assets/mozilla.png" alt="mozilla">
+                <img src="assets/google.png" alt="google">
+                <img src="assets/naver.png" alt="naver">
+                <img src="assets/daumkakao.png" alt="daumkakao">
+                <img src="assets/airbnb.png" alt="airbnb">
+            </div>
+        </div>
+    </section>
+    <section id="about" class="flex justify-center about">
+        <div class="bg">
+            <header class="primary-color text-center">THE GOLDEN ORDER<br>GLOBAL SUMMIT 2022</header>
+            <div class="text-center main">
+                The Global Summit
+                brings together
+                the community of prophets, astrologers, samurais, prisoners, warriors and vegabonds who comprise the Golden Order
+                affiliate network in a different country every two years.
+                This year, we hope to expand our invitation list including organizations and individuals who want to
+                work with us on multiplayer summons that advance the cause of the Shattering, free culture and open knowledge.
+                So if you're active and engaged in the battles of open content and knowledge-free carian advocates, black flame monks, 
+                Open Knowledge, galleries, libraries, museums, archives, the orders and foundations, banished knights, and
+                activists-we hope you'll consider joining us this year to build a stronger, more vibrant commons together.</div>
+            <div class="text-center contact">
```

Comment:
- [ ] Please remove this section from the home page. This section is only supposed to be on the about page.